documents/update_ssl_certificates.md

```diff
@@ -0,0 +1,23 @@
+### undercloud
```

Comment:
You should add a title to the doc - something like '# Updating the SSL Certificates'